views/js/review/provider/scss/qtiTestReviewProvider.scss

```diff
@@ -5,3 +5,7 @@
 @import "inc/section-container";
 @import "inc/test-dimensions";
 @import 'inc/review-layout';
+
+header nav ul li a[tabindex]:focus, .focused {
```

Comment:
Please avoid to rely on tags. Always prefer classes.
Moreover, please scope the rules to the content belonging to the component, not the entire page.

Reply:
removed